source/scripting/World.cs

```diff
@@ -687,7 +687,7 @@ public static Ped GetClosestPed(Vector3 position, float radius, params Model[] m
 		/// <summary>
 		/// A fast way to get the total number of vehicles spawned in the world.
 		/// </summary>
-		public static int VehicleCount { get { return MemoryAccess.GetNumberOfVehicles(); } }
+		public static uint VehicleCount => MemoryAccess.GetNumberOfVehicles();
```

Comment:
Sure you want this to be an `uint`? I would generally agree, but since C# is really bitchy about implcitily casting between `int` and `uint` and most iterators in C# use `int` I'm not sure it's the best choice here.

Reply:
I considered this return value type and concluded entity count in GTA world is unlikely to be above `Int32.MaxValue` (and certainly unlikely to be above 20000). Will revert it.